src/app/containers/SocialEmbed/enrichTweet.jsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react';
+import { shape } from 'prop-types';
+import socialEmbedBlockPropTypes from '#models/propTypes/socialEmbed';
+
+const EnrichTweet = ({ children }) => {
```

Comment:
Is there a need for this to be in a separate module? The three line callback within the useEffect is the only logic needed to load the widget after it's been rendered, and this code isn't shared.

Reply:
Are you suggesting moving it to the index.jsx instead @simonsinclair ?

Reply:
I would suggest moving it to the SocialEmbedContainer, yeah. In my opinion, it's this that should be calling `twittr.load()`. It would make sense to separate this useEffect if it were shared.